code/modules/uplink/uplink_items.dm

```diff
@@ -1335,47 +1443,179 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 	exclude_modes = list()
 
-/datum/uplink_item/suits/hardsuit/shielded
-	name = "Shielded Syndicate Hardsuit"
-	desc = "An upgraded version of the standard Syndicate hardsuit. It features a built-in energy shielding system. \
-			The shields can handle up to three impacts within a short duration and will rapidly recharge while not under fire."
-	item = /obj/item/clothing/suit/space/hardsuit/shielded/syndi
-	cost = 30
-	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-	exclude_modes = list()
+/datum/uplink_item/suits/hardsuit/shielded
+	name = "Shielded Syndicate Hardsuit"
+	desc = "An improved version of the standard Syndicate hardsuit. It features an advanced built-in energy shielding system. \
+			The shields can handle up to three impacts within a short duration and will rapidly recharge while not under fire."
+	item = /obj/item/clothing/suit/space/hardsuit/shielded/syndi
+	cost = 30
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+	exclude_modes = list()
+
+
+// Tools
+/datum/uplink_item/tools
+	category = "Mil-Spec Tools"
+
+/datum/uplink_item/tools/thermal
+	name = "Chameleon Thermal Imaging Glasses"
+	desc = "These goggles can be turned to resemble common eyewear found throughout the station. \
+			They allow you to see organisms through walls by capturing the upper portion of the infrared light spectrum, \
+			emitted as heat and light by objects. Hotter objects, such as warm bodies, cybernetic organisms \
+			and artificial intelligence cores emit more of this light than cooler objects like walls and airlocks."
+	item = /obj/item/clothing/glasses/thermal/syndi
+	cost = 4
+
+/datum/uplink_item/tools/nightvision
+	name = "Night Vision Goggles"
+	desc = "Simple goggles that allow one to see in the dark. Incredibly useful for infiltration."
+	item = /obj/item/clothing/glasses/night
+	cost = 2
+
+/datum/uplink_item/device_tools/magboots
+	name = "Blood-Red Magboots"
+	desc = "A pair of magnetic boots with a Syndicate paintjob that assist with freer movement in space or on-station \
+			during gravitational generator failures. These reverse-engineered knockoffs of Nanotrasen's \
+			'Advanced Magboots' slow you down in simulated-gravity environments much like the standard issue variety."
+	item = /obj/item/clothing/shoes/magboots/syndie
+	cost = 2
+	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/stealthy_tools/combatbananashoes
+	name = "Combat Banana Shoes"
+	desc = "While making the wearer immune to most slipping attacks like regular combat clown shoes, these shoes \
+		can generate a large number of synthetic banana peels as the wearer walks, slipping up would-be pursuers. They also \
+		squeak significantly louder."
+	item = /obj/item/clothing/shoes/clown_shoes/banana_shoes/combat
+	cost = 6
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear/clown_ops)
+
+/datum/uplink_item/tools/military_belt
```

Comment:
Restricted to nukies for a reason

Reply:
Chest rig was never restricted to nukies. Grenadier belt was, but I have removed the nades from it, as a new grenadier kit will be made (for each mode) instead.